clusterman/batch/cluster_metrics_collector.py

```diff
@@ -99,7 +100,11 @@ def configure_initial(self) -> None:
         # first to load the cluster config path, and then read all the entries in that directory
         self.pools: MutableMapping[str, List[str]] = {}
         for scheduler in {'mesos', 'kubernetes'}:
-            self.pools[scheduler] = get_pool_name_list(self.options.cluster, scheduler)
+            try:
+                self.pools[scheduler] = get_pool_name_list(self.options.cluster, scheduler)
+            except ClusterNotFoundError as e:
```

Comment:
How is this any different than what would happen before?  In particular it seems like we're hiding the stack trace, which is maybe valuable?  

`logger.exception` would still print the stack trace, but also the uncaught exception would print the stack trace, so I'm not sure why we need to catch it here and then raise `SystemExit`

Reply:
I was trying to hide the stack trace yes to print a more human friendly message. I can just let it through.